app/react/portainer/environments/wizard/EnvironmentsCreationView/WizardDocker/APITab/TLSFieldset.tsx

```diff
@@ -87,10 +97,18 @@ function certValidation() {
   });
 }
 
-export function validation() {
+function validation() {
```

Comment:
why not merge both validation functions?

Reply:
fixed